src/leetCodeExecutor.ts

```diff
@@ -100,6 +110,19 @@ class LeetCodeExecutor {
         }
     }
 
+    public async getCompaniesAndTags(): Promise<{ companies: { [key: string]: string[] }, tags: { [key: string]: string[] } }> {
+        // preprocess the plugin source
+        const componiesTagsPath: string = path.join(await leetCodeExecutor.getLeetCodeRootPath(), "lib", "plugins", "company.js");
+        const componiesTagsSrc: string = (await fs.readFileSync(componiesTagsPath, "utf8")).replace(
+            "module.exports = plugin",
+            "module.exports = { COMPONIES, TAGS }",
+        );
+        // require plugin from modified string
+        const requireFromString: (src: string, path: string) => any = require("require-from-string");
```

Comment:
Can we import the module?

For example, `npm i --save-dev @types/require-from-string`
Then import it instead of using require

Reply:
Yeah, the `definitelyTyped` indeed provides type for `require-from-string`.
`import * as requireFromString from "require-from-string";` successfully worked.